prepline_general/api/general.py

```diff
@@ -367,6 +369,14 @@ def pipeline_api(
         int(m_max_characters[0]) if m_max_characters and m_max_characters[0].isdigit() else 1500
     )
 
+    extract_image_block_types = (
+        json.loads(m_extract_image_block_types[0])
+        if m_extract_image_block_types and len(m_extract_image_block_types)
+        else None
+    )
+
+    extract_image_block_to_payload = bool(extract_image_block_types)
```

Comment:
I just want to double-check that `extract_image_block_to_payload` should always be `True` if `extract_image_block_types` is defined. In the unstructured repo I'm not seeing a boolean check like this and `extract_image_block_to_payload` is defaulting to `False` in all of the function definitions.

Reply:
@Coniferish This is only required when extracting image blocks via API. When users extract image blocks using the unstructured library directly on local, they can set `extract_image_block_to_payload` to True (return images stored in metadata fields)  or False (save images directly to the local file system)